examples/nlohmann_json/CMakeLists.txt

```diff
@@ -6,10 +6,7 @@ include("../common.cmake")
 
 project(dowload-nlohmann-json)
 
-hunter_add_package(nlohmann-json)
-find_package(nlohmann-json)
+hunter_add_package(nlohmann_json)
+find_package(nlohmann_json CONFIG REQUIRED)
 
 add_executable(main main.cpp)
-
-target_link_libraries(main
```

Comment:
Should we have `target_link_libraries(main nlohmann_json::nlohmann_json)` somewhere?